### DISCUSSION TITLE
Spotify - Natural changes

### DISCUSSION THREAD

**Discussion by catlikehana** [ShankarBUS/ModernFlyouts#1285] [Ideas]
Please add the ability to see modernflyouts when the spotify player naturally transitions to the next song.
Currently it only shows up when using physical media keys on a keyboard,

vers. 0.9.3.0

**Comment by picklenebula** [comment]
I would really like to see this feature if it is possible, I like using modern flyout as a sort of replacement for the windows notification system.
